bin/github-backup

```diff
@@ -356,10 +356,9 @@ def _request_http_error(exc, auth, errors):
         print('Exceeded rate limit of {} requests; waiting {} seconds to reset'.format(limit, delta),  # noqa
               file=sys.stderr)
 
-        ratelimit_error = 'No more requests remaining'
```

Comment:
This makes us ignore request limit errors when a user *is* authenticated, which I think we should actually handle?